website/app/styles/doc-components/badge.scss

```diff
@@ -28,27 +39,48 @@
   background-color: var(--doc-color-gray-200);
 }
 
+.doc-badge--type-neutral-inverted {
+  color: var(--doc-color-black);
+  background-color: var(--doc-color-gray-500);
+}
+
 .doc-badge--type-information {
   color: var(--doc-color-feedback-information-400);
   background-color: var(--doc-color-feedback-information-100);
 }
 
+.doc-badge--type-information-inverted {
+  color: var(--doc-color-feedback-information-100);
+  background-color: var(--doc-color-feedback-information-300);
```

Comment:
Apologies, I made a mistake on the color. I changed this to information-400 instead of 300 to match consistency between all badges.

Updated it here too: https://www.figma.com/design/Ky0qWjvHZR3je1lCBlzNB1/branch/S5KlbLljmucWHMTalrStWl/HDS-Website-%5BInitial-Build%5D?m=auto&node-id=826-64401&t=FP8NNiBbJWPYqSAz-1

Reply:
Fixed.

Don't forget to switch the naming for `filled` and `inverted` (see the comment I've left you in Figma)